docs/website/content/v0.3/en/components/apid.md

```diff
@@ -0,0 +1,47 @@
+---
+title: 'apid'
+---
+
+When interacting with Talos, the gRPC api endpoint you will interact with directly is apid.
+Apid acts as the gateway for all component interactions.
+Apid provides a mechanism to route requests to the appropriate destination.
+
+We'll use some examples below to illustrate what apid is doing.
+
+When a user wants to interact with a Talos component via `osctl`, there are two flags that control the interaction with apid.
+The `-e | --endpoints` flag is used to denote which Talos node ( via apid ) should handle the connection.
+Typically this is a public facing server.
+*Note* Typically the endpoints are configured in the Talos config file
```

Comment:
Notes should use `> Note: ...`. We have some CSS for `>` specifically. In docs we also put notes after the paragraph. We might also want to do something like:

```
> Note: The endpoints, and nodes can be specified in the Talos config file (e.g. `osctl config endpoints asdf`).
```

Or maybe each command is worthy of an example?